test/Xunit.Xml.TestLogger.NetCore.Tests/Xunit.Xml.TestLogger.NetCore.Tests.csproj

```diff
@@ -2,14 +2,14 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp1.0</TargetFramework>
-    <Version Condition=" '$(LoggerVersion)' == '' ">1.0.0-dev</Version>
+    <Version Condition=" '$(LoggerVersion)' == '' ">2.0.0</Version>
```

Comment:
We will extract it later into a global props file.

Reply:
Yeah.